index.bs

```diff
@@ -2018,6 +2035,37 @@ ISSUE: properly define the "finishing issuance" operation.
 ISSUE: properly define "begin redemption" operation. Consider running the algorithm at report sending time.
 
 1. Return the result of [=forgiving-base64 encoding=] |redeemedBytes|.
+    
+To <dfn>parse lookback windows</dfn> given an [=ordered map=] |map|:
+
+1. Let |lookbackWindows| be a new [=list=].
+1. If |map|["`lookback_windows`"] does not [=map/exist=], return |aggregatableDedupKeys|.
+1. Let |values| be |map|["`lookback_windows`"].
+1. If |values| is not a [=list=], return null.
+1. [=list/iterate|For each=] |value| of |values|:
+    1. If |value| is not an [=ordered map=], return null.
+    1. If |value|["`window`"] does not [=map/exist=] or is not a [=string=], return null.
+    1. Let |window| be the result of running
+        [=parse an optional 64-bit unsigned integer=] with |value|,
+        "`window`", and null.
+    1. If |window| is an error, return null.
+    1. Let |filters| be a new [=filter map=].
+    1. If |values|["`filters`"] [=map/exists=]:
+        1. Set |filters| to the result of running [=parse filter data=] with |value|["`filters`"].
+        1. If |filters| is null, return null.
+    1. Let |negatedFilters| be a new [=filter map=].
+    1. If |values|["`not_filters`"] [=map/exists=]:
+        1. Set |negatedFilters| to the result of running [=parse filter data=] with |value|["`not_filters`"].
```

Comment:
These should use https://wicg.github.io/attribution-reporting-api/#parse-filters not "parse filter data".